src/haz3lcore/statics/Statics.re

```diff
@@ -168,6 +168,15 @@ let typ_after_fix = (mode: Typ.mode, self: Typ.self): Typ.t =>
   | NotInHole(AnaInternalInconsistent(ty_ana, _)) => ty_ana
   };
 
+/* Type of a type annotation; set to unknown if it's
+   a free type variable. */
+let typ_ann_after_fix = (self: Typ.self): Typ.t =>
```

Comment:
this is kind of a hack? probably want to add UnboundVar to UTyp and have it behave like Unknown semantically (but errors still report the identifier inside)